7a_temp_coop_munge/src/data_parsers/parse_norfork_files.R

```diff
@@ -1,3 +1,88 @@
+# Parse NorkfolkReservoir_AR_monthlyTempDO files ----------------------
+parse_NorkfolkReservoir_AR_monthlyTempDO <- function(inind, outind) {
+  infile <- scipiper::sc_retrieve(inind, remake_file = '6_temp_coop_fetch_tasks.yml')
+  outfile <- scipiper::as_data_file(outind)
+
+  dat_raw <- readxl::read_xlsx(infile) %>%
+    dplyr::arrange(Timestamp)
+
+  clean <- dat_raw %>%
+    dplyr::arrange(Timestamp) %>%
+    dplyr::select(Timestamp, `Temperature (C)`, `Unit ID`) %>%
+    dplyr::mutate(DateTime = as.Date(Timestamp),
+                  time = strftime(Timestamp, format = '%H:%M', tz = 'GMT'),
+                  Norfork_ID = paste0('Norfork_', `Unit ID`)) %>%
+    dplyr::rename(temp = `Temperature (C)`) %>%
+    dplyr::select(DateTime, time, temp, Norfork_ID)
+
+  # depth is manually assigned in the spreadsheet - not all files in the raw
+  # not all files in the raw data make it into
+  # the profile analysis
+  clean <- remove_incomplete_profiles(clean)
+  clean$depth <- add_depth_data(clean)
+  clean$depth <- convert_ft_to_m(clean$depth)
```

Comment:
`tidyverse` version of this could be:

```r
clean <- clean %>% 
  remove_incomplete_profiles() %>% 
  mutate(depth = add_depth_data(.)) %>% 
  mutate(depth = convert_ft_to_m(depth))
```

Reply:
I love this enough to make the change, rebuild, and commit the files before merging this PR. This is what I was trying to do, but for some reason I struggled to break it down correctly.